examples/memory_agentic_chatbot/README.md

```diff
@@ -1,2 +1,122 @@
-This directory contains examples of how to use the `wise-agents` multi-agent framework.
 
+# Wise Agents Example: Memory Agentic Chatbot
+
+This guide walks you through running a practical example of a multi-agent system using Wise Agents. In this example, two agents—a web interface agent and an intelligent agent—are started, allowing you to experiment with agent communication and interaction in a simulated environment.
+
+
+## Example Overview
+
+The example consists of two main agents:
+
+1. **Web Interface Agent**: Simulates a web-based client for interacting with other agents.
+2. **Intelligent Agent**: Handles requests and provides intelligent responses based on memory and context.
+
+These agents are defined in YAML configuration files located in the `examples/memory_agentic_chatbot` directory.
+
+## Running the Example
+
+### Step 1: Clone the Repository
```

Comment:
Just a comment for later. I imagine many of these steps will likely be repeated across examples so maybe later we can create a directory with common snippets so they can be included in other examples.